src/python/WMComponent/DBS3Buffer/DBSBufferBlock.py

```diff
@@ -211,14 +215,14 @@ def addBlockParent(self, parent):
         self.data['block_parent_list'].append({'block_name': parent})
         return
 
-    def addDatasetParent(self, parent):
+    def addDatasetParents(self, parents):
```

Comment:
Why is it a list if there is only one parent dataset? I'd suggest to change it to singular too